tools/scripts/transform.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Should we have transformation as a separate step (from generation)?
Sounds like a good idea although we need to update GitHub action with an extra step (I can help with that in separate PR)

Reply:
Sure sounds good! I can try to add it on this PR